app/lib/frontend/oauth2_service.dart

```diff
@@ -36,9 +40,15 @@ class OAuth2Service {
     try {
       final api = new oauth2_v2.Oauth2Api(authClient);
       info = await api.userinfo.get();
+      return info.email;
+    } on AccessDeniedException catch (e, st) {
+      _logger.log(
+          Level.INFO, 'Bearer token lookup failed with access denied.', e, st);
```

Comment:
I would rephrase that as 'Access denied for bearer token'. What's a bearer token btw, maybe mention oauth or something?